scripts/pull.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# (C) 2022 GoodData Corporation
+import os, shutil, tempfile
+from gooddata_sdk import GoodDataSdk
+from pathlib import Path
+from dotenv import load_dotenv
+
+# TODO "gooddata_layouts" should be imported from the lib
+LAYOUT_ROOT_FOLDER = "gooddata_layouts"
+
+def main():
+    # Get current environment, defaulting to "development"
+    environment = os.getenv("GD_ENV", "development")
+
+    # Load corresponding env variables from file, if exists
+    load_dotenv(f".env.{environment}")
+
+    # Get all the necessary arguments from environment
+    host = os.getenv("GD_HOST")
+    if host is None:
+        raise RuntimeError("GD_HOST environmental variable must be defined")
+
+    token = os.getenv("GD_TOKEN")
+    if token is None:
+        raise RuntimeError("GD_TOKEN environmental variable must be defined")
+
```

Comment:
I would suggest doing something more generic here and reuse it in pull.py and push.py. GD_HOST and GD_TOKEN are mandatory. So let's do the following:

```
import os

def get_mandatory_env_var(key: str) -> str:
    try:
        return os.environ[key]
    except KeyError:
        raise RuntimeError(f"{key} environmental variable must be defined")

host = get_mandatory_env_var("GD_HOST")
token = get_mandatory_env_var("GD_TOKEN")
```